code/game/machinery/computer/fax_responder_spy.dm

```diff
@@ -0,0 +1,329 @@
+/obj/structure/machinery/computer/spy_camera
+	name = "remote monitoring computer"
+
+	icon_state = "terminal"
+
+	var/mob/hologram/spy_camera/spy_eye
+	var/spy_range = 5
+	var/spy_faction = FACTION_NEUTRAL
+
+	var/turf/last_location
+	var/turf/start_location
+
+	/// Computer and Spycam can only be used if this variable is cleared
+	var/locked = FALSE
+
+/obj/structure/machinery/computer/spy_camera/attackby(obj/I as obj, mob/user as mob)  //Can't break or disassemble.
```

Comment:
make sure not to use abbreviations (no mob/M), 
Also why use `as obj` and `as mob` here? AFAIK these are only really useful for verbs.